src/neptune/types/atoms/file.py

```diff
@@ -174,15 +174,16 @@ def from_stream(stream: IOBase, *, seek: Optional[int] = 0, extension: Optional[
     def as_image(image, autoscale: bool = True) -> "File":
         """Static method for converting image objects or image-like objects to an image File value object.
 
-        This way you can upload `Matplotlib` figures, `Seaborn` figures, `PIL` images, `NumPy` arrays, as static images.
+        This way you can upload Matplotlib figures, Seaborn figures, PIL images, and NumPy arrays as static images.
 
         Args:
             image: Image-like object to be converted.
```

Comment:
We should mention that the input image pixel should be either in `[0.0, 1.0]`(floats)  or `[0-255]` (integer). In case they have something weird they need to do the normalization on their end.